tests/chapter3-base-tests.md

```diff
@@ -79,6 +81,10 @@ rather a mock font that has faked metrics.
     >>> test_layout("<big><big><i>abc</i></big>def</big>") #doctest: +NORMALIZE_WHITESPACE
     [(13.0, 22.5, 'abc', Font size=24 weight=normal slant=italic style=None), 
      (109.0, 25.5, 'def', Font size=20 weight=normal slant=roman style=None)]
+     
+    >>> test_layout("hello world")
+    [(13.0, 22.5, 'hello', Font size=16 weight=normal slant=roman style=None),
+	(13.0, 42.5, 'world', Font size=16 weight=normal slant=roman style=None)]
```

Comment:
Whitespace is wrong here, tests will not pass. You need to either:
- Put the list on one line (don't do this)
- Change `NORMALIZE_WHITESPACE` (don't do this either)
- Use `wbemocks.print_list` (do this one)